packages/arb-token-bridge-ui/src/components/common/SearchPanel/SearchPanelTable.tsx

```diff
@@ -0,0 +1,67 @@
+import { MagnifyingGlassIcon } from '@heroicons/react/24/outline'
+import { AutoSizer, List, ListRowRenderer } from 'react-virtualized'
+import React from 'react'
+
+export const SearchPanelTable = ({
+  searchInputPlaceholder,
+  searchInputValue,
+  SearchFieldCTA,
```

Comment:
nit consistency
```suggestion
  searchFieldCta,
```

Reply:
i did that because it's a React Node
seems a bit strange either way

Reply:
Can we at least do: `SearchFieldCta`? We agreed to have script PascalCase even for abbreviations